forms.py

```diff
@@ -0,0 +1,50 @@
+from flask_wtf import Form
+from wtforms import (StringField, BooleanField, PasswordField,
+                    TextAreaField)
+from wtforms.validators import (DataRequired, Regexp, ValidationError, 
+                                Email, Length, EqualTo)
+
+from models import User
+
+def email_exists(form, field):
+    if User.select().where(User.email == field.data).exists():
+        raise ValidationError('User with that email already exists.')
+
+class RegisterForm(Form):
+    email = StringField(
+        'Email',
+        validators=[
+            DataRequired(),
+            Email(),
+            email_exists
+        ])
+    password = PasswordField(
+        'Password',
+        validators=[
+            DataRequired(),
+            Length(min=8),
+            EqualTo('password2', message='Passwords must match.')
+        ])
+    password2 = PasswordField(
```

Comment:
as @tabiodun suggested on my Repo, you may want to consider renaming this variable to something more descriptive like `confirm_password`.